src/psyclone/tests/psyir/symbols/datasymbol_test.py

```diff
@@ -41,12 +41,16 @@
 from __future__ import absolute_import
 import pytest
 
+from fparser.common.readfortran import FortranStringReader
```

Comment:
Update copyright date and remove absolute_import

Reply:
Done.